docs/diagrams/SearchCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,83 @@
+@startuml
+/' Sequence Diagram for postal sector search '/
+!include style.puml
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":OrderBookParser" as OrderBookParser LOGIC_COLOR
+participant ":SearchCommandParser" as SearchCommandParser LOGIC_COLOR
+participant ":ArgumentTokenizer" as ArgumentTokenizer LOGIC_COLOR
```

Comment:
The methods implemented in ArgumentTokenizer are clearly and correctly reflected in the diagram. However, to make the sequence diagram clearer and more readable, the process of tokenizing and mapping might be omitted. 😄